utils/summon.cfg

```diff
@@ -91,34 +97,32 @@ For more information, choose the 'Summoning Help' rightclick option which appear
             [/store_gold]
             {IF_VAR actualgold greater_than_equal_to {COST} (
                 [then]
-                    [gold]
-                        amount=-{COST}
-                        side=$side_number
-                    [/gold]
-                    [unit]
-                        type={UNIT}
-                        side=$side_number
-                        x,y=$x1,$y1
-                        moves=0
-                        animate=yes
-                    [/unit]
-                    {MODIFY_UNIT x,y=$x1,$y1 attacks_left 0}
-                    {MODIFY_UNIT x,y=$x1,$y1 moves 0}
+                    [sync_variable]
```

Comment:
I am not familiar with sync_variable so just relying on your testing of you changes. Nothing immediately and strongly wrong noticed.